src/internal/mod.rs

```diff
@@ -4,6 +4,7 @@
 
 use crate::internal::{
     rest::{Authorization, IronCoreRequest},
+    user_api::DeviceId,
```

Comment:
Don't you want this merged with the next line?

Reply:
Fixed by cargo fmt